### PR TITLE
[FIX] Report Translations and TZ issue

### PR DESCRIPTION
**Pull request by eilst**
[FIX] Report Translations and TZ issue

### CODE REVIEW

File: project_event/report/project_task_activity_report.xml [hunk 1]
```diff
@@ -84,14 +73,14 @@
...
-              <th>Expected Departure</th>
+              <th>Real Arrival</th>
             </tr>
             <t t-foreach="tasks" t-as="task">
               <tr>
                 <td> <t t-esc="task['department']"/></td>
                 <td> <t t-esc="task['employee']"/></td>
                 <td> <t t-esc="datetime.datetime.strptime(task['expected_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
-                <td> <t t-esc="datetime.datetime.strptime(task['expected_departure'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
+                <td> <t t-esc="datetime.datetime.strptime(task['real_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
```
Comment (rimbendhaou) [inline project_event/report/project_task_activity_report.xml:83]: real_start is not in tasks, it is in task_details

File: project_event/report/project_task_activity_report.xml [hunk 2]
```diff
@@ -43,37 +39,30 @@
       <div style="border-bottom: 2px solid black">
         <table class="table">
           <tr>
-            <td> <t t-esc="activity_label"/> <t t-esc="name"/>  </td>
+            <td>Activity: <t t-esc="name"/>  </td>
```
Comment (rimbendhaou) [inline project_event/report/project_task_activity_report.xml:42]: Activity is not translated

File: project_event/report/project_task_activity_report.xml [hunk 3]
```diff
@@ -43,37 +39,30 @@
...
+            <td width="50%">Date: <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%Y-%m-%d')"/>
             </td>
-            <td width="50%"> <t t-esc="number_spectators_label"/></td>
+            <td width="50%">Number of spectators: </td>
             </tr>
             <tr>
-              <td> 
-                <t t-esc="start_hour_label"/>
-                <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/>              
+              <td>Hour: <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/>
```
Comment (rimbendhaou) [inline project_event/report/project_task_activity_report.xml:50]: It didn't take the summer hour

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/report/project_task_activity_report.xml (hunk 1 above)
Comment (eilst) [reply project_event/report/project_task_activity_report.xml:83]: Right, added to tasks

File: project_event/report/project_task_activity_report.xml (hunk 2 above)
Comment (eilst) [reply project_event/report/project_task_activity_report.xml:42]: Done

File: project_event/report/project_task_activity_report.xml (hunk 3 above)
Comment (eilst) [reply project_event/report/project_task_activity_report.xml:50]: I do not see this problem

Comment (eilst) [reply project_event/report/project_task_activity_report.xml:50]: The TZ that you chose on user preferences should be the one you see

**Review by rimbendhaou — APPROVED** [review]

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**